docs/contributing.md

```diff
@@ -8,7 +8,7 @@ If you already cloned the repository and you know that you need to deep dive in
 
 ### Python
 
-SQLModel supports Python 3.6 and above, but for development you should have at least **Python 3.7**.
+SQLModel supports Python 3.8 and above, but for development you should have at least **Python 3.7**.
```

Comment:
This is not correct, since before it said for development, you need 3.7, which was higher than the usage of requirement of 3.6. So now, you should just remove that part.